### PR TITLE
Add set command and add get info by display name

### LINKED ISSUE #35: User config command
In the original spreadsheet we had custom display names and icons for which languages were being learned. Introducing the languages field and making these attributes configurable would be a good next step for customization. It can later be built on with badges and cumulative words.

Thinking the command could be along the lines of `.set`. `.set display_name "Toast"` and `.set languages DE EN`

### PR DESCRIPTION
**Pull request by azhitian**
Closes #35

You can set your display name with `.set name 😸:angery:🥰:shibthumbsup:` and your languages with `.set languages 🇹🇼🇹🇼🇹🇼`

Info now also shows an `@` mention (without pinging) of the user and their languages

Since display names will be in the leaderboard, it makes sense to be able to search for user info by them. We could also include @ mentions in the leaderboard, but I feel like this defeats the purpose of setting your own display name

### CODE REVIEW

File: src/word_debt_bot/cogs/game_commands.py [hunk 1]
```diff
@@ -94,21 +94,75 @@ async def info(
...
+            self.journal(
+                {
+                    "command": "set",
+                    "user": str(ctx.author.id),
+                    "key": key,
+                    "value": value,
+                }
+            )
+        elif key.lower() == "languages":
+            if len(value) > 16:
```
Comment (Swiddis) [inline src/word_debt_bot/cogs/game_commands.py:148]: Is a limit of 16 too strict for languages? Some people are learning a lot of them...

**Review by Swiddis — APPROVED** [review]
Minor nit

File: src/word_debt_bot/cogs/game_commands.py (hunk 1 above)
Comment (azhitian) [reply src/word_debt_bot/cogs/game_commands.py:148]: I thought the idea was we'd be using flags or abbreviations and they'd be displayed in certain places (could add them to the leaderboard, for example), so having a bunch would be inconvenient

But I don't care that much and the Welsh flag is like length 9 so I'll bump it up

Comment (azhitian) [reply src/word_debt_bot/cogs/game_commands.py:148]: We could chop it off where it's displayed anyway as well

**Comment by azhitian** [comment]
Addressed feedback

**Pull request merged**